diesel_cli/tests/print_schema.rs

```diff
@@ -31,8 +31,16 @@ fn run_infer_schema_exclude() {
 }
 
 #[test]
-fn run_infer_schema_order() {
-    test_print_schema("print_schema_order", vec!["--with-docs"]);
+fn run_infer_schema_table_order() {
+    test_print_schema("print_schema_table_order", vec!["--with-docs"]);
+}
+
+#[test]
+fn run_infer_schema_column_order() {
+    test_print_schema(
+        "print_schema_column_order",
+        vec!["--column-sorting", "name"],
```

Comment:
We pass sorting related arguments here and set them also in the config file below. I think we should have two separate tests for those cases.

Reply:
That's how all the other tests do it as well. We would need to refactor `test_print_schema` to be able to split up config and argument tests (since `test_print_schema` calls the config test after testing the arguments).